wikimedia/wikipedia-ios
# [ALT TEXT] Remove main alt-text developer settings flag

**Phabricator:** https://phabricator.wikimedia.org/T370362

### Notes
* Removes the main `Enable Alt Text experiment` dev settings flag
* Other flags are still available 

### Test Steps
1. Run the app with an account with less than 50 edits
2. Verify other dev settings flags are off
3. Add one of the target languages (pt-br, es, fr, zh) 
4. Verify you can see the suggested edits card on the explore feed
5. Add a non-target language as your main language
6. Verify the suggested edits card is no longer available
